### PR TITLE
IBX-6407 Add missing entries to bundles reference

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6407
| Versions      | master
| Edition       | N/A

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

File: docs/administration/project_organization/bundles.md [hunk 1]
```diff
@@ -46,57 +46,72 @@ see the [How to Remove a Bundle]([[= symfony_doc =]]/bundles/remove.html) instru
...
 |Bundle|Description|
 |---------|-----------|
-|[ibexa/support-tools](https://github.com/ibexa/support-tools)|System information|
-|[ibexa/admin-ui-assets](https://github.com/ibexa/admin-ui-assets)|Assets for the Back Office|
 |[ibexa/admin-ui](https://github.com/ibexa/admin-ui)|Back Office interface|
+|[ibexa/admin-ui-assets](https://github.com/ibexa/admin-ui-assets)|Assets for the Back Office|
 |[ibexa/content-forms](https://github.com/ibexa/content-forms)|Form-based integration for the Symfony Forms into Content and User objects in kernel|
+|[ibexa/core](https://github.com/ibexa/core)|Core of the [[= product_name =]] application|
+|[ibexa/core-persistence](https://github.com/ibexa/core-persistence)|Core system persistence|
 |[ibexa/core-extensions](https://github.com/ibexa/core-extensions)|Core system functionalities|
```
Comment (adamwojs) [inline docs/administration/project_organization/bundles.md:54]: Cane we remove link to ibexa/core-extensions ? There is no such package.

File: docs/administration/project_organization/bundles.md [hunk 2]
```diff
@@ -106,9 +121,13 @@ see the [How to Remove a Bundle]([[= symfony_doc =]]/bundles/remove.html) instru
 
 |Bundle|Description|
 |---------|-----------|
+|ibexa/experience|Ibexa Experience packages|
```
Comment (adamwojs) [inline docs/administration/project_organization/bundles.md:124]:
```suggestion
|ibexa/experience|Metapackage for Symfony Flex-based Ibexa DXP Experience installation|
```

File: docs/administration/project_organization/bundles.md [hunk 3]
```diff
@@ -46,57 +46,72 @@ see the [How to Remove a Bundle]([[= symfony_doc =]]/bundles/remove.html) instru
...
+|ibexa/commerce-fieldtypes|Shop-specific Field Types **deprecated** |
+|ibexa/commerce-price-engine|Engine for handling prices **deprecated** |
+|ibexa/commerce-shop-ui|UI for the shop front page **deprecated** |
+|ibexa/commerce-shop|Main shop functionalities **deprecated** |
 
 ## Ibexa Experience packages
 
 |Bundle|Description|
 |---------|-----------|
+|ibexa/headless|Ibexa Headless packages|
```
Comment (adamwojs) [inline docs/administration/project_organization/bundles.md:109]:
```suggestion
|ibexa/headless|Metapackage for Symfony Flex-based Ibexa DXP Headless installation|
```

Comment (MagdalenaZuba) [reply docs/administration/project_organization/bundles.md:109]:
```suggestion
|ibexa/headless|Metapackage for Symfony Flex-based [[= product_name =]] Headless installation|
```

**Review by adriendupuis — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**